pkg/proto/aksnodeconfig/v1/helper.go

```diff
@@ -48,38 +48,38 @@ func ensureConfigsNonNil(nBC *Configuration) {
 	initializeIfNil(&nBC.CustomSearchDomainConfig)
 }
 
-// NewNBContractBuilder creates a new instance of NBContractBuilder and ensures all objects in nodeBootstrapConfig are non-nil.
-func NewNBContractBuilder() *NBContractBuilder {
-	nbc := Configuration{
+// NewAKSNodeConfigBuilder creates a new instance of AKSNodeConfigBuilder and ensures all objects in aKSNodeConfig are non-nil.
+func NewAKSNodeConfigBuilder() *AKSNodeConfigBuilder {
+	config := Configuration{
 		Version: contractVersion,
 	}
-	ensureConfigsNonNil(&nbc)
-	nBCB := &NBContractBuilder{nodeBootstrapConfig: &nbc}
-	return nBCB
+	ensureConfigsNonNil(&config)
+	aKSNodeConfig := &AKSNodeConfigBuilder{aKSNodeConfig: &config}
+	return aKSNodeConfig
 }
 
-// ApplyConfiguration Applies the configuration to the nodeBootstrapConfig object.
-func (nBCB *NBContractBuilder) ApplyConfiguration(config *Configuration) {
+// ApplyConfiguration Applies the configuration to the aKSNodeConfig object.
+func (aKSNodeConfigBuilder *AKSNodeConfigBuilder) ApplyConfiguration(config *Configuration) {
```

Comment:
nit: i think in general it is better for the receiver name to be concise e.g. `b *AKSNodeConfigBuilder` like Cameron suggested on my other PR.

Reply:
Sounds good. Updated.